geometry_msgs/PoseStamped graspPose
---
bool result
bool earlyFailureDetected
